steps/init_env.sh

```diff
@@ -17,6 +17,8 @@
 # init_env.sh
 set -eE -x
 
+echo -e "${RED}this script is deprecated, please run 'make thirdparty' in top directory instead${NC}"
```

Comment:
delete this unused file directly?

Reply:
https://github.com/4paradigm/OpenMLDB/blob/main/.github/workflows/integration-test-src.yml not upgraded @wuyou10206